src/Neo/SmartContract/Manifest/ContractEventDescriptor.cs

```diff
@@ -77,5 +77,13 @@ public virtual JObject ToJson()
             json["parameters"] = new JArray(Parameters.Select(u => u.ToJson()).ToArray());
             return json;
         }
+
+        public bool Equals(ContractEventDescriptor other)
+        {
+            if (other == null) return false;
```

Comment:
you don't need this line, ReferenceEquals covers it. Only do in the override of `Equals`

Reply:
Done